academic-services/knowledge-exploration-service/reference-makes-command-line-tool.md

```diff
@@ -70,43 +75,63 @@ The base URL to a MAKES release package.
 
 The MAKES host virtual machine image resource Id. Run CreateHostResource command to generate one.
 
-### Optional parameters
+### DeployHost optional parameters
 
 `--MakesIndex`
 
-The MAKES index files URL reference.
+The MAKES index files URL reference. Defaults to "<MakesPackage>/index".
 
 `--MakesGrammar`
 
-The MAKES grammar file URL reference.
+The MAKES grammar file URL reference. Defaults to "<MakesPackage>/grammar/makes-default-grammar".
 
 `--Region`
 
-The region where the MAKES API host should be deployed to.
+The region where the MAKES API host should be deployed to. Defaults to "westus"
 
 `--InstanceCount`
 
-The default number of MAKES API host instances (virtual machines).
+The default number of MAKES API host instances (virtual machines). Defaults to 1.
 
 `--HostMachineSku`
 
-The Azure Virtual Machine SKU for MAKES API host machines.
+The Sku for MAKES API host machines. Check [Azure Virtual Machine Sizes](https://docs.microsoft.com/azure/virtual-machines/windows/sizes) to get the avaliable options. Defaults to "Standard_D14_v2".
 
-## CreateIndexResources Command
+`--HostMachineDataDiskSizeInGb`
 
-Creates the MAKES index building resources. E.g. MAKES index build batch account, storage account.
+The size of the data disk (Managed Disk:Premium SSD) that the host should have. See [Azure Managed Disks](https://azure.microsoft.com/pricing/details/managed-disks/) for detailed pricing information. If the size is set to 0, no additional data disk will be attached to the host Virtual Machine and index/grammar data will be stored on the temp drive (D:\\) that comes with the Virtual Machine. If the size is great than 0, then we'll add a managed disk with the specified size, intialize as data disk drive (F:\\) and download the index/grammar data to it. Note: Using data disk instead of the default attached temp drive on the Virtual Machine may lead to decrease performance due to IOPS differences. Defaults to 0.
```

Comment:
>Note: Using data disk instead of the default attached temp drive on the Virtual Machine may lead to decrease performance due to IOPS differences [](start = 557, length = 144)

Should we use the pre-formatted 'note' bubble for this?